eos-usb-reflash/factory-test/start.sh

```diff
@@ -6,6 +6,30 @@
 set -e
 mydir=$(dirname $(readlink -f $0))
 
+# If this script is running from an exFAT filesystem we need to manually load
+# the exfat kernel module, as on EOS prior to 4.0 the USB device has been
+# mounted using the FUSE exFAT driver, and the initramfs we are going to pivot
+# to does not contain an exFAT driver (neither in-kernel nor via FUSE).
+FSTYPES=$(lsblk --noheadings --output FSTYPE)
```

Comment:
Do we care if there is an exFAT filesystem on a device other than `$mydir`? Here you're checking all devices, but I think only the one that holds the initramfs matters. Would it work to find the mountpoint for `$mydir` and run `$lsblk` against that? (I guess in practice there's no difference.)

Reply:
Oh… is `$mydir` the directory in `/var` that everything's already been unpacked to, so actually we don't know what USB device will need to be mounted?

Reply:
> Oh… is `$mydir` the directory in `/var` that everything's already been unpacked to, so actually we don't know what USB device will need to be mounted?

Exactly! That is why I am checking if any exfat fs is attached to the system, I think it is a good enough heuristic in this situation.